embulk-jruby-strptime/src/main/java/org/jruby/lexer/StrptimeLexer.java

```diff
@@ -0,0 +1,603 @@
+/* The following code was generated by JFlex 1.4 on 4/21/17 2:55 PM */
```

Comment:
We don't need to put the .java file. I will take a look how to generate this file later. This file is generated with JFlex 1.4 that JRuby uses. Since the version is old, it's not supported by several JFlex gradle plugin but.